prometheus/generic/setup-ilo-exporter/tasks/prereqs.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Ensure epel-release is installed
+  yum:
```

Comment:
Probably should call out the dependency in the README as this step will most likely fail on some hosts, i.e.: `RHEL` hosts. Otherwise, perhaps add some tasks above to set up the requires yum channels automatically if not already in place.